app/Managers/PostManager.php

```diff
@@ -72,9 +72,10 @@ public function create($data, $signupId, $authenticatedUserRole = null)
      *
      * @param \Rogue\Models\Post $post
      * @param array $data
+     * @param bool $log
      * @return \Rogue\Models\Post
      */
-    public function update($post, $data)
+    public function update($post, $data, $log)
```

Comment:
I think we should give this a default value. `$log = true`

Reply:
ah right - forgot that!